docs/pages/linking/overview.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Overview of Linking, Deep Links, Android App Links, and iOS Universal Links
+sidebar_title: Overview
+description: An overview of available resources to implement Linking and Deep Links in your Expo apps.
+---
+
+import { AndroidIcon } from '@expo/styleguide-icons/custom/AndroidIcon';
+import { AppleIcon } from '@expo/styleguide-icons/custom/AppleIcon';
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CODE } from '~/ui/components/Text';
+
+## Linking
+
+Linking allows your app to interact with incoming and outgoing URLs. In this process, the user not only gets directed to open your app, but they are taken to a specific screen (route) within the app.
+
+For example, by clicking an advertisement of your product, your app will open on the user's device and they can view that product's details. This product's link that the user clicked may look like:
+
+```html
+myapp://web-app.com/product
+```
+
+This link is constructed by three parts:
+
+- **Scheme**: The URL scheme that identifies the app that should open the URL (example: `myapp://`). It can also be `https` or `http` for non-standard deep links.
+- **Host**: The domain name of the app that should open the URL (example: `web-app.com`).
+- **Path**: The path to the screen that should be opened (example: `/product`). If the path isn't specified, the user is taken to the home screen of the app.
+
+## Linking strategies
+
+There are different linking strategies you handle in your Expo app:
+
+- Linking to other apps from your app (outgoing links)
+- Linking to your app from other apps or a web browser (incoming links)
+- Android App Links
+- iOS Universal Links
+
+> **info** **Tip:** We recommend using [Development builds](/develop/development-builds/introduction/) to test your app's linking strategies instead of Expo Go.
+
+### Into other apps
+
+Linking to other apps from your app is achieved using a URL based on the target app's URL scheme. This **URL scheme** allows you to reference resources within that native app.
+
+Your app can use a [common URL scheme](/linking/into-other-apps/#common-url-schemes) for default apps or implement any URL scheme, including `https` and `http` (commonly used by web browsers like Chrome, Safari, and so on), and use JavaScript to handle the URL that launches the corresponding native app.
+
+<BoxLink
+  title="Linking into other apps"
+  description="Learn how to handle common and custom URL schemes to link other apps from your app."
+  href="/linking/into-other-apps/"
+  Icon={BookOpen02Icon}
+/>
+
+### Into your app
+
+Linking to your app from other apps or a web browser is achieved using a URL based on the scheme for your app. These URLs are called **deep links**.
```

Comment:
The document introduces "links" and "deep links" in the title. This is the first time afterwards where it references "deep links". If we're presenting these as two concepts (which I think is useful on an overview page like this), let's differentiate them more strongly. I think this definition is a good one to anchor this to: https://en.wikipedia.org/wiki/Deep_linking. Deep links aren't specific to apps; they're just links to specific url-based content inside of a website or app. 

Thinking back to apps, any app with a registered scheme can be linked to (e.g., `myapp://whatever`). Only apps that listen for deep links and perform routing based on them really implement "deep linking", and it just happens that Expo Router makes this happen automatically. (seems like long ago, but a lot of web apps used to not support deep linking... actually, many still don't).